apps/signal/51-function-call-effect/src/app/user.service.ts

```diff
@@ -1,10 +1,15 @@
-import { Injectable, signal } from '@angular/core';
+import { Injectable, signal, untracked } from '@angular/core';
 
 @Injectable({ providedIn: 'root' })
 export class UserService {
   name = signal('Thomas');
 
+  // name is considered a dependency by Angular
+
   log(message: string) {
-    console.log(`${this.name()}: ${message}`);
+    console.log(untracked(() => this.name()) + ':' + message);
```

Comment:
I need to update the documentation. Let's say, userService is a third party library, we don't have access to it.